src/core/webview/webviewMessageHandler.ts

```diff
@@ -2106,6 +2106,7 @@ export const webviewMessageHandler = async (
 				}
 
 				await provider.upsertProviderProfile(message.text, configToSave)
+				vscode.commands.executeCommand("kilo-code.ghost.reload") // kilocode_change: Reload ghost model when API provider settings change
```

Comment:
note to self: we need to check whether this whole command can't more easily be a JS function call.